pkg/api/v1/types.go

```diff
@@ -1715,6 +1715,16 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// NodeDaemonPorts lists ports opened by daemons running on the Node
+type NodeDaemonPorts struct {
+	// Port on which Kubelet is listening
+	KubeletPort int `json:"kubeletPort,omitempty" description:"port on which Kubelet is listening"`
```

Comment:
Remove the description tags. We're now deriving documentation from comments, as per my post to kubernetes-dev@googlegroups.com.